### PR TITLE
JA: Add ja/lite/g3doc/performance/post_training_integer_quant.ipynb

### PR DESCRIPTION
**Pull request by zinziroge**
Original Document:
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/g3doc/performance/post_training_integer_quant.ipynb

### CODE REVIEW

File: site/ja/lite/performance/post_training_integer_quant.ipynb [hunk 1]
```diff
@@ -0,0 +1,682 @@
...
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "BTC1rDAuei_1"
+      },
+      "source": [
+        "## 概要\n",
+        "\n",
+        "現在、[TensorFlow Lite](https://www.tensorflow.org/lite/) は TensorFlow から TensorFlow Lite のフラットバッファー形式へ変換する際に、\n",
```
Comment (kiszk) [inline site/ja/lite/performance/post_training_integer_quant.ipynb:76]: [weak suggestion]: [this table](https://github.com/tfug/proofreading/blob/master/terminologies.txt) uses the policy without `ー｀
```suggestion
        "現在、[TensorFlow Lite](https://www.tensorflow.org/lite/) は TensorFlow から TensorFlow Lite のフラットバッファー形式へ変換する際に、\n",
```

File: site/ja/lite/performance/post_training_integer_quant.ipynb [hunk 2]
```diff
@@ -0,0 +1,682 @@
...
+        "モデルのすべての値(重みと活性化関数)を8ビット整数へ変換することをサポートしています。\n",
```
Comment (kiszk) [inline site/ja/lite/performance/post_training_integer_quant.ipynb:77]:
```suggestion
        "モデルのすべての値(重みと活性化関数の出力)を8ビット整数へ変換することをサポートしています。\n",
```

File: site/ja/lite/performance/post_training_integer_quant.ipynb [hunk 3]
```diff
@@ -0,0 +1,682 @@
...
+        "これによりモデルの大きさが4倍小さくなり、CPU上での性能が3~4倍改善します。\n",
+        "さらに、この完全に量子化されたモデルは整数演算のみに対応したハードウェアアクセラレータで使用できます。\n",
+        "\n",
+        "重みのみを8bitで保持する [訓練後の \"オン・ザ・フライ\" 量子化](https://colab.sandbox.google.com/github/tensorflow/tensorflow/blob/master/tensorflow/lite/tutorials/post_training_quant.ipynb) とは対照的に、\n",
+        "この手法はモデル変換中に静的にすべての重み `と` 活性化関数を量子化します。\n",
```
Comment (kiszk) [inline site/ja/lite/performance/post_training_integer_quant.ipynb:82]: [weak suggestion]
```suggestion
        "この手法はモデル変換中に静的にすべての重み *と* 活性化関数の結果を量子化します。\n",
```

File: site/ja/lite/performance/post_training_integer_quant.ipynb [hunk 4]
```diff
@@ -0,0 +1,682 @@
...
+        "\n",
+        "このチュートリアルでは、MNIST モデルをスクラッチから訓練し、その精度を TensorFlow で検査しそれから、モデルを完全に量子化された TensorFlow Lite のフラットバッファーに変換します。\n",
```
Comment (kiszk) [inline site/ja/lite/performance/post_training_integer_quant.ipynb:84]: [suggestion]
```suggestion
        "このチュートリアルでは、MNIST モデルをスクラッチから訓練し、その精度を TensorFlow で検査しそれから、モデルを完全に量子化された TensorFlow Lite のフラットバッファに変換します。\n",
```

**Review by lamberta — APPROVED** [review]

**Comment by lamberta** [comment]
Thanks and sorry for the delay, @zinziroge 

As mentioned on the [mailing list](https://groups.google.com/a/tensorflow.org/g/docs/c/G23C6ZxjiUI/m/z8G87HTfBQAJ), we're directing the community to work alongside vendors in our [GitLocalize project](https://gitlocalize.com/tensorflow/docs-l10n). It's the only way to make sure the translations are in sync with the updated source material. The community can submit pull requests through GitLocalize and provide reviews.

But merging this PR since it doesn't appear any work has been started here: https://gitlocalize.com/repo/4592/ja/site/en-snapshot/lite/performance/post_training_integer_quant.ipynb
In the future please submit reviews and pull requests using the GitLocalize UI. Thank you

**Pull request merged**